### PR TITLE
add -tail to job log

### PR DESCRIPTION
**Pull request by jxr98**
**Make sure that you've checked the boxes below before you submit PR:**

### Always

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Written well with PR title, we generate the [release notes](https://github.com/jenkins-zh/jenkins-cli/releases) base on that

### For the bug fixes or features only

- [ ] [Quality Gate Passed](https://sonarcloud.io/dashboard?id=jenkins-zh_jenkins-cli). Change this URL to your PR.
- [ ] The coverage is xxx on the new lines
- [ ] I've tested it by manual in the following platform
  - [ ] MacOS
  - [ ] Linux
  - [x] Windows
- [ ] Unit Test covered
- [ ] e2e Test covered

### CODE REVIEW

File: app/cmd/job_log.go [hunk 1]
```diff
@@ -34,6 +34,8 @@ func init() {
 		i18n.T("Watch the job logs"))
 	jobLogCmd.Flags().IntVarP(&jobLogOption.Interval, "interval", "i", 1,
 		i18n.T("Interval of watch"))
+	jobLogCmd.Flags().IntVarP(&jobLogOption.NumberOfLines,"tail","t",-1,
+		i18n.T("the last number of lines of the log"))
```
Comment (LinuxSuRen) [inline app/cmd/job_log.go:38]: The first letter should be capital.

File: app/cmd/job_log.go [hunk 2]
```diff
@@ -44,16 +46,24 @@ It'll print the log text of the last build if you don't give the build id.`),
...
 			} else {
 				err = fmt.Errorf("job history must be a number instead of '%s'", historyStr)
 			}
+			numberOfLinesStr :=args[2]
+			numberOfLines, err:=strconv.Atoi(numberOfLinesStr)
+			if err!=nil||numberOfLines<=0 {
+				err =fmt.Errorf(err.Error(),"lines of job must be a positive integer instead of '%s'",numberOfLinesStr)
+			}else{
+				jobLogOption.NumberOfLines=numberOfLines
+			}
```
Comment (LinuxSuRen) [inline app/cmd/job_log.go:66]: It's not necessary to parse the value of `tail`. The following code can do that.

`jobLogCmd.Flags().IntVarP(&jobLogOption.NumberOfLines,"tail","t",-1,
		i18n.T("the last number of lines of the log"))`

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Thanks for your contribution. The calculation seems wrong. Please take a look at it.

```
➜  jenkins-cli git:(tail) ✗ ./jcli job log test -t 2
Current build number: 1
Current build url: http://139.198.3.176:30180/job/test/1/
Finished: SUCCESS
➜  jenkins-cli git:(tail) ✗ ./jcli job log test -t 3
Current build number: 1
Current build url: http://139.198.3.176:30180/job/test/1/
[Pipeline] End of Pipeline
Finished: SUCCESS
```

Please make sure that you have executed the command: `make fmt` before you submit.

File: app/cmd/job_log.go (hunk 2 above)
Comment (jxr98) [reply app/cmd/job_log.go]: All righty. I've changed that. The lines of 'Current build number' and 'Current build url' are not counted in the 'tail number', but there was a bug about the number of lines to be printed and it has been fixed. Thx.

**Review by LinuxSuRen — APPROVED** [review]
It works well. Thanks!

```
➜  jenkins-cli git:(tail) jcli job log test -t 7
Current build number: 1
Current build url: http://139.198.3.176:30180/job/test/1/
[Pipeline] // stage
[Pipeline] }
[Pipeline] // withEnv
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
Finished: SUCCESS
```

**Comment by jxr98** [comment]
You're welcome. Thx for reviewing my code.

**Pull request merged**